src/main/java/de/slimecloud/slimeball/features/staff/absence/AbsenceTable.java

```diff
@@ -0,0 +1,30 @@
+package de.slimecloud.slimeball.features.staff.absence;
+
+import de.mineking.databaseutils.Table;
+import de.mineking.databaseutils.Where;
+import de.mineking.discordutils.list.Listable;
+import net.dv8tion.jda.api.entities.UserSnowflake;
+
+import java.time.Instant;
+import java.util.List;
+import java.util.Optional;
+
+public interface AbsenceTable extends Table<Absence>, Listable<Absence> {
```

Comment:
Listable is not correctly implemented. Adding the interface is not enough. Take a lok at other listable tabled to see what methods are required and how they should generally look.

Reply:
do I need it at all?

Reply:
You don't *need* it, but I think it would be helpful to br able to see als absences with the provided details